tests/acceptance/test_update.py

```diff
@@ -461,6 +488,13 @@ def test_signed_updates(self, sig_case, bitbake_path, bitbake_variables):
                                 os.remove(data_file)
                         finally:
                             os.chdir("..")
+                    if not sig_case.header_checksum_ok:
+                        # Corrupt the header checksum (the same as corrupting header itself)
```

Comment:
Actually, it's not, because if you change the checksum, then the signature check fails, but this we already know. What we need to do is to keep the checksum the same, so that the signature check passes, but change the actual header, which would fail when matching the checksum against the file.